packages/gatsby/index.d.ts

```diff
@@ -932,32 +933,58 @@ export interface Store {
   replaceReducer: Function
 }
 
-type logMessageType = (format: string, ...args: any[]) => void
-type logErrorType = (message: string, error?: Error) => void
+type LogMessageType = (format: string, ...args: any[]) => void
```

Comment:
Args here are actually wrong, those methods on reporter (`success`, `info`, `warn`, `log`, `verbose`) only get single string argument.

Reply:
Good spot. I just capitalised the name and didn't follow it through but you're right!

Reply:
Oh, yup! You are not introducing wrong args here - it just clicked when reviewing PR that what we currently have is wrong